packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -577,11 +577,11 @@ getCollectionHooks("Posts").updateAsync.add(async function updatedPostMaybeTrigg
 
   // If the old document was a draft and the new document is not, or the author is no longer unreviewed, trigger the webhook
   if ((oldDocument.draft && !document.authorIsUnreviewed) || (oldDocument.authorIsUnreviewed && !document.authorIsUnreviewed)) {
-    await callType3Webhook('post_published', url);
+    void callType3Webhook('post_published', url);
     return
   }
 
   if (oldDocument.contents?.html !== document.contents?.html) {
-    await callType3Webhook('post_edited', url);
+    void callType3Webhook('post_edited', url);
```

Comment:
I think raw reading costs are nontrivial and we could cut them my a factor of >2 by checking for the size of the diff.

Reply:
Couldn't this be handled on type 3's end? It seems simpler to always tell them when there is an update

Reply:
Seems reasonable